rails/app/views/nursery_tables/show.html.erb

```diff
@@ -15,5 +15,36 @@
   <%= @nursery_table.name %>
 </p>
 
+<p>
+  Recent restoration activities:
+</p>
+
+<table class="table">
+    <thead>
+      <tr>
+        <th scope="col">Date Performed</th>
+        <th scope="col">Cleaned?</th>
+        <th scope="col">Percent full</th>
+        <th scope="col">Bleached</th>
+        <th scope="col">Dead</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @nursery_table.restoration_activity_log_entries.order('id DESC').each do |entry| %>
```

Comment:
should we include restoration_activity_log_entries in the initial query to avoid n+1s?

Reply:
Hi @LizPrescott would it be possible to give me an example please?